src/converter.js

```diff
@@ -18,9 +18,20 @@ const DEFINITION_PREFIX = 'Definition';
 const DROP_ATTRIBUTE_MARKER = Symbol('A marker to drop the attributes');
 
 const referencePrefix = '#/definitions/';
+
+function normalizeTypeName (typeName) {
+  const removedExtension = typeName.replace('.json', '').replace('schema', '');
```

Comment:
I find this line hard to understand. Would you consider replacing just suffixes with empty string, like `typeName.replace(/\.json$/, '')` and why is the `schema` word being removed?

Reply:
I agree, this can be refactored.  Looking at some of these [JSON Schema examples](https://json-schema.org/learn/miscellaneous-examples.html), I noticed that these `$id` keys pointed to files that had a _.schema.json_ extension.  This code is intended to remove the _.schema_ as well, but I don't think most users will have that format, so it's probably not needed.  The goal is to extract the part of the string that is between the last `/` and the `.json` extension.  How about this?
```
const normalizedTypeName = uppercamelcase(
   typeName
      .slice(typeName.lastIndexOf('/') + 1, typeName.length')
      .replace(/\.json$/, '');
)
```
If a user puts in a URI, this will extract the file name, but if the string is already a valid GraphQL type, it will be unchanged.

Reply:
I like your proposed approach. If you wanted to get really fancy to handle the `.schema` behavior, you could try this regular expression `/(\.schema)?\.json$/` to match either `.schema.json` or `.json`

Reply:
Hmm, I like the fancy regex!  I'll try this out to make sure it works as expected.